plugin/ochttp/trace.go

```diff
@@ -66,6 +66,11 @@ func (t *traceTransport) RoundTrip(req *http.Request) (*http.Response, error) {
 	}
 
 	if t.format != nil {
+		header := make(http.Header)
```

Comment:
Copy paste the request cloning from https://github.com/golang/oauth2/blob/master/transport.go#L108:6.

Reply:
I intentionally did not copy the Request struct itself because it would already have been copied by the WithContext calls directly above this. I added a comment to this effect.